src/js/languages/default.json

```diff
@@ -80,8 +80,8 @@
     "required": "required"
   },
   "formField": {
-    "characters": {
-      "left": {
+    "maxCharacters": {
+      "remaining": {
```

Comment:
Changing keys like this isn't backwards compatible and probably constitutes a breaking change

Reply:
These keys haven't been committed yet